command/base/command.go

```diff
@@ -198,6 +211,10 @@ func printTitle(w io.Writer, s string) {
 func printFlag(w io.Writer, f *flag.Flag) {
 	example, _ := flag.UnquoteUsage(f)
 	if example != "" {
+		// Change 'value' to 'string' for consistency
+		if example == "value" {
+			example = "string"
+		}
```

Comment:
Hmm - we should probably keep the default Go behavior here for consistency. Looking at `UnquoteUsage` you can fix `-http-addr` by putting `string` in backquotes in the help text:

```
The `address` to connect to...
```

This will make it do `-http-addr=<address>` and is probably a better way to customize. While in here I realized that the description for `-http-addr` doesn't address UNIX sockets at all, so we need to touch that up a bit.